diagnostic.md

```diff
@@ -45,7 +45,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+_It runs two times. x is 4 after while loop_
 
```

Comment:
Incorrect. Always feel free to test code like this in node or a repl.

It runs 3 times, and the value is 8.